src/CommandingOfficers/Tech.java

```diff
@@ -214,29 +212,41 @@ protected void perform(MapMaster gameMap)
     @Override
     public GameEventQueue getEvents(MapMaster gameMap)
     {
-      boolean log = false;
+      boolean log = true;
       if( log ) System.out.println("[TechDrop] getEvents() entry");
 
       // Prep events for `numDrops` deployments. Recalculate landing position between each.
       Set<XYCoord> dropLocs = new HashSet<XYCoord>();
       for( int i = 0; i < numDrops; ++i )
       {
-        TeleportEvent techDrop = generateDropEvent(gameMap, dropLocs, log);
+        TeleportEvent techDrop = generateDropEvent(gameMap, log);
         if( null != techDrop )
         {
-          dropLocs.add(techDrop.getEndPoint());
+          // Store the new event.
           abilityEvents.add(techDrop);
+
+          // Place it on the map temporarily so we account for it in subsequent drops.
+          dropLocs.add(techDrop.getEndPoint());
+          Unit u = techDrop.getUnit();
+          XYCoord loc = techDrop.getEndPoint();
+          gameMap.moveUnit(u, loc.xCoord, loc.yCoord);
```

Comment:
This... this feels like a hack.

Reply:
Total hack, and it was breaking things. Fixed.